src/Base/Exception.cpp

```diff
@@ -24,12 +24,9 @@
 
 #include "PreCompiled.h"
 
-#include <typeinfo>

```

Comment:
typeid() is used in Exception.cpp

Reply:
OK, but now you committed it with the deletion: https://github.com/FreeCAD/FreeCAD/commit/89b37a5110fbcc54248e7b9061713b685051442c

Reply:
It's included by PyObjectBase.h